src/app/model/Mission.ts

```diff
@@ -384,6 +384,21 @@ class Mission extends BaseModel {
       status: MissionStatus.tentative,
     });
   }
+
+  /**
+   * Submit feedback for mission
+   * @param {string} missionUid : mission for which feedback will be set
+   * @param {string} feedback: mission feedback
+   * @param {boolean} success: whether the the mission was a success.
+   */
+  submitFeedback(missionUid: string, feedback: string, success: boolean) {
+    //TODO: update firestore.rules to allow only if the mission is created by that user
+    return this.update(missionUid, {
+      feedbackNotes: feedback || "",
+      ...(success && { status: MissionStatus.succeeded }),
```

Comment:
Let me know if this short-hand seems hacky. Found it [here](https://stackoverflow.com/questions/50367085/assign-and-set-js-object-property-conditionally)

Reply:
interesting. A little unorthodox, but I think it's fine for this PR.